[2024-12-05T06:18:28.600+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T06:18:28.618+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_pipeline.save_to_postgres manual__2024-12-05T06:18:25.393011+00:00 [queued]>
[2024-12-05T06:18:28.627+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_pipeline.save_to_postgres manual__2024-12-05T06:18:25.393011+00:00 [queued]>
[2024-12-05T06:18:28.628+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-05T06:18:28.641+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): save_to_postgres> on 2024-12-05 06:18:25.393011+00:00
[2024-12-05T06:18:28.647+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3076) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T06:18:28.649+0000] {standard_task_runner.py:72} INFO - Started process 3077 to run task
[2024-12-05T06:18:28.649+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_pipeline', 'save_to_postgres', 'manual__2024-12-05T06:18:25.393011+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline.py', '--cfg-path', '/tmp/tmpekl88fkw']
[2024-12-05T06:18:28.651+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask save_to_postgres
[2024-12-05T06:18:28.698+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_pipeline.save_to_postgres manual__2024-12-05T06:18:25.393011+00:00 [running]> on host b86987f6f159
[2024-12-05T06:18:28.800+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Dung' AIRFLOW_CTX_DAG_ID='weather_pipeline' AIRFLOW_CTX_TASK_ID='save_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T06:18:25.393011+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T06:18:25.393011+00:00'
[2024-12-05T06:18:28.801+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T06:18:28.805+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-05T06:18:28.817+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T06:18:28.818+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_pipeline, task_id=save_to_postgres, run_id=manual__2024-12-05T06:18:25.393011+00:00, execution_date=20241205T061825, start_date=20241205T061828, end_date=20241205T061828
[2024-12-05T06:18:28.866+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T06:18:28.882+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-05T06:18:28.884+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
